accepted/2022/ReflectionInvoke.md

```diff
@@ -0,0 +1,474 @@
+# Reflection Invoke for 8.0 (draft \ in progress)
+#### Steve Harter
+#### April 13, 2023
+
+# Background
+For additional context, see:
+- [Developers using reflection invoke should be able to use ref struct](https://github.com/dotnet/runtime/issues/45152)
+- [Originally library issue; needs refreshing\updating](https://github.com/dotnet/runtime/issues/10057)
```

Comment:
Good read overall. Though I found the use of "\\" instead of "/" jarring
```suggestion
- [Originally library issue; needs refreshing/updating](https://github.com/dotnet/runtime/issues/10057)
```

Ditto other places.